webapp-django/crashstats/profile/templates/profile/profile.html

```diff
@@ -0,0 +1,198 @@
+{% extends "crashstats_base.html" %}
+
+{% block page_title %}Your Profile{% endblock %}
+
+{% block site_css %}
+    {{ super() }}
+
+    {% compress css %}
+<link type="text/css" rel="stylesheet" href="{{ static('profile/css/profile.css') }}">
+    {% endcompress %}
+
+{% endblock %}
+
+{% block site_js %}
+    {{ super() }}
+
+<script src="{{ static('crashstats/js/moment.min.js') }}"></script>
+
+    {% compress js %}
+<script type="text/javascript" src="{{ static('crashstats/js/socorro/your_crashes.js') }}"></script>
```

Comment:
Yeah it's being used in the ``your crashes`` page, though as that one is doomed to disappear, I wouldn't bother optimizing there and compress this file with others.